### PR TITLE
feat: Suppress "Add detour" dropdown on mobile

### PR DESCRIPTION
**Pull request by joshlarson**
Asana Ticket: https://app.asana.com/0/1148853526253420/1207448960080664/f

### CODE REVIEW

File: assets/src/components/routeLadder.tsx [hunk 1]
```diff
@@ -213,6 +214,11 @@ const RouteLadder = ({
 
   const displayCrowding = someVehicleHasCrowding(vehiclesAndGhosts, route.id)
 
+  const screenSize = useScreenSize()
```
Comment (firestack) [inline assets/src/components/routeLadder.tsx:217]: can we do this within CSS instead?

Comment (joshlarson) [reply assets/src/components/routeLadder.tsx:217]: We could - I think that would make it a bit more difficult to test, but perhaps that's okay?

Comment (firestack) [reply assets/src/components/routeLadder.tsx:217]: Yeah I'm not really sure we'd be able to test it until we have CSS in Jest tests, but that's fine with me. We could assert a certain class is on it, I think we just need to change it to `d-none d-sm-block`, or something like that https://getbootstrap.com/docs/5.3/utilities/display/ (see the `Hidden only on xs` text)

Comment (joshlarson) [reply assets/src/components/routeLadder.tsx]: @firestack I updated to match ☝️, but I'm not sure that I love the lack of testability. I'll let it sit with me over the evening.

Comment (joshlarson) [reply assets/src/components/routeLadder.tsx]: @firestack I've let it sit with me, and while I don't _love_ relying on snapshot tests, they will solve this issue.

Ready for re-review!

**Review by firestack — APPROVED** [review]

**Pull request merged**